demos/printerdemo/node/package.json

```diff
@@ -4,9 +4,10 @@
     "main": "main.js",
     "type": "module",
     "dependencies": {
-        "slint-ui": "../../../api/node"
+        "slint-ui": "workspace:*"
     },
     "scripts": {
-        "start": "node ."
+        "start": "node .",
+        "prestart": "cd ../../../api/node/ && node build-on-demand.mjs && pnpm compile"
```

Comment:
does that mean user needs to install pnpn to run the demo? is that not a problem?

Reply:
That ppm dependency exists already today :)

Reply:
I guess the docs need update: https://github.com/slint-ui/slint/tree/master/demos#running-the-node-examples

I find it a bit back that the examples require extra non-standard dependencies though.

Reply:
Indeed, I'll adapt the docs.

This was a conscious decision when we introduced pnpm though, didn't we? We're choosing the productivity of slint development itself over the ability and ease of running the examples/demos.

We could have a separate discussion to see if it would make sense if our examples and demos would depend on the nightly snapshots of Python and Node packages and install from there. I wouldn't mind.